src/app/screens/History/ExpressionContainer/index.js

```diff
@@ -1,24 +1,71 @@
-import React from 'react';
-import {View, Text, ImageBackground, TouchableOpacity} from 'react-native';
+import React, {useState} from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  ImageBackground,
+  TouchableOpacity,
+} from 'react-native';
+import {connect} from 'react-redux';
```

Comment:
You can also use hooks:

`const dispatch = useDispatch()`

and then you can get items as

`const history = useSelector(store => store.historyReducer.history)`

Reply:
I understand, I was reading the hooks, they are not complicated to implement, I already made the changes.